lib/metadata/index.js

```diff
@@ -29,18 +30,16 @@ function inferDownstreamReview(metadata, content) {
         return "Firefox";
     }
 
-    if (content.indexOf("WebKit export") > -1 &&
-        (login == "fwang" || login == "ms2ger" || login == "rniwa" ||
-        login == "youennf")) {
+    if (metadata.isWebKitVerified && metadata.webkit.flags.reviewed) {
         return "WebKit";
     }
-
     return null;
 }
 
-module.exports = function getMetadada(number, author, content) {
+module.exports = function getMetadada(number, author, title, content) {
```

Comment:
I'm not sure if all callsites to this have been updated. For example, in `test/get-metadata.js`, shouldn't every call to `getMetadata` have been updated? (As otherwise they are now passing in their previous content for the title, and `undefined` for the content).

Can you verify everywhere that pulls in `lib/metadata` and uses it is passing the right arguments?